cdap-common/src/main/java/co/cask/cdap/common/HttpExceptionHandler.java

```diff
@@ -64,13 +64,13 @@ public void handle(Throwable t, HttpRequest request, HttpResponder responder) {
       responder.sendString(HttpResponseStatus.NOT_IMPLEMENTED, t.getMessage());
     } else {
       LOG.error("Unexpected error: request={} {} user={}:", request.getMethod().getName(), request.getUri(),
-                SecurityRequestContext.getUserId().or("<null>"), t);
+                SecurityRequestContext.getUserId(), t);
```

Comment:
How do we differentiate between a user ID of `null` and `"null"` in the log message? (Although previously, there was no way to differentiate between a user ID of `null` and `"<null>"`.)